lib/meta_tags/meta_tags_collection.rb

```diff
@@ -128,8 +128,11 @@ def extract_separator
     def extract_noindex
       noindex_name,  noindex_value  = extract_noindex_attribute(:noindex)
       nofollow_name, nofollow_value = extract_noindex_attribute(:nofollow)
+      follow_name, follow_value = extract_noindex_attribute(:follow)
 
-      if noindex_name == nofollow_name
+      if noindex_name == follow_name && (follow_value && noindex_value) 
+        { noindex_name => [noindex_value, follow_value].compact.join(', ') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.